backend/src/main/java/de/adrianaschepers/shishabuddies/model/SetupEntity.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Die Klasse ist komplett auskommentiert -> müsste es in diesem Commit eigentlich nicht dabei sein, da sie ja gar nicht verwendet wird.